### ISSUE TITLE
Cannot login to cluster when user home is mounted as network drive

### ISSUE THREAD

**Issue by sbouchet** [redhat-developer/intellij-openshift-connector#147]
[kind/bug]

## What versions of software are you using?
- Operating System: Windows
- IntelliJ version: 2019.3
- OpenShift version: odo 1.0.2

## Actual behavior
This is a issue created to follow the discussion on gitter for a user that cannot log in the cluster using a network drive as home folder.
The openshift view list the cluster but not the projects.
the cluster tree is empty

using odo in commandline ( odo login ) works.

user screenshot : https://files.gitter.im/redhat-developer/openshift-connector/Yuel/image.png

## Expected behavior
user can be able to login to the cluster.

## Any logs, error output, etc?
There is no output/error neither in the UI or the logs.

**Comment by doublefx** [comment]
Thanks @sbouchet for the time you spent with me.

I managed to get the cluster list but not the projects after I found an initial workaround:

- Creating a windows link into my `%USERPROFILE%` (`c:\Users\<ME>`) pointing to `%HOMESHARE%%HOMEPATH%\.kube` (`H:\.kube`).
- Creating a new `%USER_HOME%` environment variable with `%USERPROFILE%` as value.

Therefore giving the user the option to set the .kube's lookup path from the connector would be a good starting point.

There might be some other issues as:

- I don't get the project list.
- `FileSystems.getDefault().newWatchService()` won't work on a mapped network drive.
